# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1711561550 -3600
# Parent  306080b2e3e52de24c9bac23c61b4881efe0c9f8
Misc: Set even attribute on even tree rows for use in userChrome.css

diff --git a/mail/base/content/widgets/tree-view.mjs b/mail/base/content/widgets/tree-view.mjs
--- a/mail/base/content/widgets/tree-view.mjs
+++ b/mail/base/content/widgets/tree-view.mjs
@@ -2678,8 +2678,11 @@ export class TreeViewTableRow extends HT
         : "option"
     );
     this.setAttribute("aria-posinset", index + 1);
     this.id = `${this.list.id}-row${index}`;
+    if (index % 2 === 0) {
+      this.setAttribute("even", "true");
+    }
 
     const isGroup = this.view.isContainer(index);
     this.classList.toggle("children", isGroup);
 
